includes/class-scheduler.php

```diff
@@ -199,6 +205,39 @@ public static function reprocess_outbox() {
 		}
 	}
 
+	/**
+	 * Purge outbox items based on a schedule.
+	 */
+	public static function purge_outbox() {
+		$total_posts = (int) wp_count_posts( Outbox::POST_TYPE )->publish;
+		if ( $total_posts <= 20 ) {
```

Comment:
should we use a const for the number?

Reply:
Let's keep it simple for now and see if we get feedback about it?